packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx

```diff
@@ -19,9 +19,28 @@ const defaultProps = {
   src: ''
 };
 
-const Avatar = ({ className, src, alt, ...props }) => (
-  <img {...props} className={css(_styles.avatar, className)} src={src} alt={alt} />
-);
+/**
+ * Column properties.
+ */
+export interface IAvatarProps {
```

Comment:
I am not super opinionated about this, but I know most other typescript libs have dropped prefixing interfaces with `I` it was kind of a holdout from C#/Java, and most people in the JS/TS community prefer leaving it off.

Reply:
Changed it to AvatarProps